citi-onboarding/virtu-onepage
# Feature/our services section

### What I did
- Made the OurServices Section work responsively.
- Created an auxiliar commponent called CardSerice, to display the cards in OurServicesSection
- Changed some image SVG files to correspond to what I was doing with the icons inside cards
- In tablet size, it was not explicit in the template, but I scaled up the cards to look better

### How to test
1. Enter client folder with `cd client`
2. Execute the command `yarn` to download the project modules
3. Execute the command `yarn start` to start the server
3. Open your browser on `http://localhost:3000/`
4. See if the OurServices sections made is responsive and equivalent to the mockup

Resolves #20
